openstack/sharedfilesystems/v2/sharenetworks/urls.go

```diff
@@ -9,3 +9,7 @@ func createURL(c *gophercloud.ServiceClient) string {
 func deleteURL(c *gophercloud.ServiceClient, id string) string {
 	return c.ServiceURL("share-networks", id)
 }
+
+func listURL(c *gophercloud.ServiceClient) string {
+	return c.ServiceURL("share-networks", "detail")
```

Comment:
There is both operations but I chose to implement only one as we are planning to only use this one. Would you like me to implement the other one, too ?